packages/cashscript/test/Instance.test.ts

```diff
@@ -27,6 +30,19 @@ describe('Instance', () => {
     });
   });
 
+  describe('Contract Utxo inputs', () => {
```

Comment:
It's difficult to test this really well so this is good 👍

Reply:
removed as private methods prevent testing